vcam-util.c

```diff
@@ -183,28 +188,29 @@ int main(int argc, char *argv[])
         case 'r':
             current_action = ACTION_DESTROY;
             dev.idx = atoi(optarg) - 1;
+            printf("Removing the device.\n");
```

Comment:
I have change the execute sequence to unify the output logic.
Which we declare the assignment task in `main`, and print additional `stderr` when the task failed.